dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -71,13 +75,19 @@ internal class RumSessionScope(
 
     // region Internal
 
-    private fun onApplicationDisplayed(
+    internal fun onApplicationDisplayed(
         event: RumRawEvent.StartView,
         viewScope: RumViewScope,
         writer: Writer<RumEvent>
     ) {
-        applicationDisplayed = true
-        viewScope.handleEvent(RumRawEvent.ApplicationStarted(event.eventTime), writer)
+        if (!applicationDisplayed) {
+            applicationDisplayed = true
```

Comment:
isn't `applicationDisplayed { return viewScope is not nil }` since if `viewScope` is set that means app has a view therefore is displayed? why do we need to manually set it?

Reply:
We nullify the `ViewScope` on iOS, but AFAIR it's different on Android - we keep the `ViewScope` object alive and we reset its state.

Reply:
oh now i remember we talked about this 👌